frontend/src/components/Layout.tsx

```
@@ -157,29 +157,29 @@ export default function Layout() {
           >
             <FileText className="w-5 h-5" />
             API Docs
           </a>
           {/* #9: Keyboard shortcut button */}
           <button
-            onClick={() => window.dispatchEvent(new KeyboardEvent('keydown', { key: '?' }))}
-            className="flex items-center gap-3 px-3 py-2.5 rounded-lg text-sm font-medium text-gray-500 dark:text-slate-400 hover:text-gray-900 dark:hover:text-white hover:bg-gray-100 dark:bg-space-800 transition-colors focus-ring w-full"
+            onClick={() => globalThis.dispatchEvent(new KeyboardEvent('keydown', { key: '?' }))}
+            className="flex items-center gap-3 px-3 py-2.5 rounded-lg text-sm font-medium text-gray-500 dark:text-slate-400 hover:text-gray-900 dark:hover:text-white hover:bg-gray-100 dark:hover:bg-space-800 transition-colors focus-ring w-full"
             aria-label="Keyboard shortcuts"
           >
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)